src/resmom/start_exec.c

```diff
@@ -2851,6 +2851,27 @@ receive_job_update_request(int sd)
 	}
 }
 
+/**
+ * @brief	work task handler for end of a job in mock run mode
+ *
+ * @param[in]	ptask - pointer to the work task
+ *
+ * @return void
+ */
+static void
+mock_run_end_job_task(struct work_task *ptask)
+{
+	job *pjob;
+
+	pjob = ptask->wt_parm1;
```

Comment:
Will it not crash in the cases where a job is deleted and this task triggers after the job is deleted? Or is the mom already deleting all the pending tasks on the basis of param1?

Reply:
@agrawalravi90 I didn't see a change for this comment. I am ok if you want to keep it as is because this mode is just for testing purposes. Let me know.